others/environment.py

```diff
@@ -123,39 +131,3 @@ def set_computation_library(self, ComputationLib: "type[ComputationLibrary]"):
     
     def set_logs(self, logs: "dict[str, Any]"):
         self._logs = logs
-
-    # Overloading properties/methods for Bharadhwaj implementation
-    @property
-    def a_size(self):
-        return self.action_space.shape[0]
-
-    def reset_state(self, batch_size):
-        self.reset()
-
-    @property
-    def B(self):
-        return self._batch_size
-
-    def rollout(self, actions, return_traj=False):
-        # Uncoditional action sequence rollout
-        # actions: shape: TxBxA (time, batch, action)
-        assert actions.dim() == 3
-        assert actions.size(1) == self.B, "{}, {}".format(actions.size(1), self.B)
-        assert actions.size(2) == self.a_size
-        T = actions.size(0)
-        rs = []
-        ss = []
-
-        total_r = torch.zeros(self.B, requires_grad=True, device=actions.device)
-        for i in range(T):
-            # Reshape for step function: BxTxA
-            s, r, _, _ = self.step(actions[i])
-            rs.append(r)
-            ss.append(s)
-            total_r = total_r + r
-            # if(done):
-            #     break
-        if return_traj:
-            return rs, ss
-        else:
-            return total_r
```

Comment:
Removed some old methods when I tried adapting to the original CEM+Grad implementation. We're not using this anymore though.